queries/ruby/highlights.scm

```diff
@@ -127,6 +124,14 @@
   (#any-of? @constant.builtin
     "__callee__" "__dir__" "__id__" "__method__" "__send__" "__ENCODING__" "__FILE__" "__LINE__"))
 
+((identifier) @keyword.function
+  (#any-of? @keyword.function
+    "include" "extend" "prepend" "attr_reader" "attr_writer" "attr_accessor" "module_function"
```

Comment:
`include` looks like it should be `keyword.import`. The other doesn't really look like functions, moreso just general keywords?

Reply:
@lucario387 in ruby "include" "extend" "prepend" "attr_reader" "attr_writer" "attr_accessor" "module_function" are not technically keywords but special class/module methods https://www.rubydoc.info/stdlib/core/Object:extend - so it's more like a function with a keyword meaning that's why i thought `@keyword.function` is the closes available group to it (also this group gives the right color)

Reply:
Got it, so include is moreso "adding things to base class", rather than typical imports. Got it

Reply:
why not `@function.builtin`?

Reply:
@amaanq currently highlights.scm doesn't include `@function.builtin` so i just don't want to introduce new highlight groups. Also `@function.builtin` doesn't return the correct color by default so it will require adjusting this highlight group color for ruby

Reply:
Colors are irrelevant here; that's a matter for individual colorschemes. Important is that the captures are _semantically_ correct. And any capture in CONTRIBUTING.md can (and should, if appropriate) be used.

Reply:
@function.builtin group has been added